migration/tests/mavenver.rs

```diff
@@ -0,0 +1,74 @@
+use migration::sea_orm::Statement;
+use migration::ConnectionTrait;
+use test_context::test_context;
+use test_log::test;
+use trustify_common::db::Database;
+use trustify_test_context::TrustifyContext;
+
+#[path = "./version_common.rs"]
+mod version_common;
+
+async fn mavenver_cmp(
+    db: &Database,
+    left: &str,
+    right: &str,
+) -> Result<Option<i32>, anyhow::Error> {
+    let result = db
+        .query_one(Statement::from_string(
+            db.get_database_backend(),
+            format!(
+                r#"
+        SELECT * FROM mavenver_cmp( '{left}', '{right}' )
+        "#,
+            ),
+        ))
+        .await?;
+
+    if let Some(result) = result {
+        Ok(result.try_get_by_index(0)?)
+    } else {
+        Ok(None)
+    }
+}
+
+#[test_context(TrustifyContext)]
+#[test(tokio::test)]
+async fn test_mavenver_cmp(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
```

Comment:
have you considered any negative tests ... though maybe some error handling on version_matches might be needed

Reply:
Such as comparing "1.0" to "tacos" ?

Reply:
ya valid input I guess ... though admittedly perhaps this is a guard rail best down higher up in version_matches